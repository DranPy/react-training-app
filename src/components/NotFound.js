import React from 'react';
import { FontAwesomeIcon as FA } from '@fortawesome/react-fontawesome';

export default () => {
  return (
    <div className="not-found">
      <div className="container">
        <h1 className="text-center">
          <FA icon="ban" size="3x" />
          <div>Page not found</div>
        </h1>
      </div>
    </div>
  );
};
