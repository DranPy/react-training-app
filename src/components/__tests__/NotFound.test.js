import { shallow } from 'enzyme';
import React from 'react';

import NotFound from '../NotFound';

describe('<NotFound />', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
