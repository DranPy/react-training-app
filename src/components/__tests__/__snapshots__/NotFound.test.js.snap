// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should match snapshot 1`] = `
<div
  className="not-found"
>
  <div
    className="container"
  >
    <h1
      className="text-center"
    >
      <FontAwesomeIcon
        border={false}
        className=""
        fixedWidth={false}
        flip={null}
        icon="ban"
        inverse={false}
        listItem={false}
        mask={null}
        pull={null}
        pulse={false}
        rotation={null}
        size="3x"
        spin={false}
        symbol={false}
        title=""
        transform={null}
      />
      <div>
        Page not found
      </div>
    </h1>
  </div>
</div>
`;
